sklearn/feature_extraction/text.py

```diff
@@ -1087,9 +1118,7 @@ def transform(self, X, copy=True):
 
     @property
     def idf_(self):
-        # if _idf_diag is not set, this will raise an attribute error,
-        # which means hasattr(self, "idf_") is False
-        return np.ravel(self._idf_diag.sum(axis=0))
+        return np.log(float(self._n_samples) / self._df) + 1.0
```

Comment:
Remove `float()` and add `division` to the `__future__` import above please. The `__future__` is now!